interface-definitions/include/firewall/firewall-hashing-parameters.xml.i

```diff
@@ -0,0 +1,35 @@
+<!-- include start from firewall/firewall-hashing-parameters.xml.i -->
+<leafNode name="hash">
+  <properties>
+    <help>Define the parameters of the packet header to apply the hashing, or choose random</help>
```

Comment:
Please remove `, or choose random` as this node has a `defaultValue`. The following string will be automatically appended when generating CLI nodes: `(default: random)`.